Changelog.md

```diff
@@ -18,6 +18,7 @@ Bugfixes:
  * Immutables: Fix wrong error when the constructor of a base contract uses ``return`` and the parent contract contains immutable variables.
  * IR Generator: Add missing cleanup during the conversion of fixed bytes types to smaller fixed bytes types.
  * IR Generator: Add missing cleanup for indexed event arguments of value type.
+ * IR Generator: Fix ICE when copying reference types in calldata and storage to struct or array members in memory.
```

Comment:
We use ICE liberally, but the term is really difficult to understand without background.

Reply:
I can fix this. There are liked 4 other ICEs in the same section! Will leave untouched in this PR.

Reply:
Fixed!